src/GameModel.cpp

```diff
@@ -541,7 +542,7 @@ namespace gameModel{
 
     auto Team::getPlayerByID(communication::messages::types::EntityId id) const -> std::optional<std::shared_ptr<Player>> {
         for(const auto &player : getAllPlayers()){
-            if(player->id == id){
+            if(player->getId()== id){
```

Comment:
ein Leerzeichen wäre schön ;)
```suggestion
            if(player->getId() == id){
```